main/main.go

```diff
@@ -133,11 +135,19 @@ func main() {
 	if logLevel <= logging.Debug {
 		networkLogLevel = logLevel
 	}
+	var trackedSubnets set.Set[ids.ID]
```

Comment:
can you add a comment why this is needs to be passed, insetad of previously in app request network